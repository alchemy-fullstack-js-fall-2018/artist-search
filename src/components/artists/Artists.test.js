// import React from 'react';
// import Artists from './Artists';
// import { shallow } from 'enzyme';

// describe('artists', () => {
//     let wrapper;

//     beforeEach(() => {
//         wrapper = shallow(<Artists />);
//         console.log('wrapper', wrapper);
//     });

//     it('renders a artists component', () => {
//         expect(wrapper).toMatchSnapshot();
//     });
// });
