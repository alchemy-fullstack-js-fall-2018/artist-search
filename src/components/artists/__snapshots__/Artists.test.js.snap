// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`artists renders a artists component 1`] = `undefined`;
